pages/common/swc.md

```diff
@@ -0,0 +1,28 @@
+# swc
+
+> JavaScript and TypeScript compiler written in Rust.
+> More information: <https://swc.rs>.
+
+- Transpile a specified input file and output to stdout:
+
+`swc {{path/to/file}}`
+
+- Transpile a specified input file and output to a specific file:
+
+`swc {{path/to/input_file}} --out-file {{path/to/output_file}}`
+
+- Transpile the input file every time it is changed:
+
+`swc {{path/to/file}} --watch`
+
+- Transpile a specified input directory and output to a specific file:
+
+`swc {{path/to/input_directory}} --config-file {{path/to/.swcrc}} --out-dir {{path/to/output_directory}}`
+
+- Transpile a specified input directory using a specific configuration file:
+
+`swc {{path/to/input_directory}} --config-file {{path/to/.swcrc}}`
```

Comment:
This example should probably be swapped with the previous one, to follow the practice of gradual complexity buildup.

Reply:
Oh, I forgot to remove `--config-file` from the previous example when I split the `--out-dir` and `--config-file` into 2 examples.